Adds newline and wrap buttons to code view

leleSahil opened the pull request:
Addresses #329 
Allows long lines code view to be wrapped in the window and/or new lines to be rendered to aid in reading output

Before<img width="959" alt="before" src="https://cloud.githubusercontent.com/assets/5100798/25026686/4eb9f932-206d-11e7-9bfc-420ee41f5762.png">

After
<img width="960" alt="after" src="https://cloud.githubusercontent.com/assets/5100798/25026691/5a1cdfe2-206d-11e7-942f-3a606288c98f.png">

estee-tew commented:
https://stackstorm.slack.com/archives/community/p1492122082018664

armab commented:
Looks useful indeed :+1:

enykeev commented on modules/st2-highlight/highlight.directive.js:
```diff
@@ -44,6 +44,8 @@ module.exports =
       scope.$watch('code', function (code) {
 
         scope.hidden = true;
+	scope.wrap = false;
```
Please make sure you use spaces and not tabs for indentation. Not sure why linter didn't point you to that, will check.

ckamps commented:
Hi @leleSahil, Is it feasible for you to include in this PR an enhancement to make the horizontal scroll bar much more prominent?

Note that in the following example where Chrome is being used, the scroll bar is very hard to pick out.  It's just to the left of the middle in this example.

![image](https://cloud.githubusercontent.com/assets/1332810/25076908/ce67aecc-2342-11e7-8ad4-bad0eab8ed4a.png)

leleSahil commented:
Hi @enykeev, I can't seem to find any good icons on fontello. Is it absolutely necessary to use an icon from there? Or to use an icon at all?

enykeev commented:
No, not really necessary, icons just help something to look actionable without using default button appearance. Just style them to look like other buttons in st2web, that should be enough.

leleSahil commented:
Hi @enykeev, I've changed the styling to match the other buttons in st2web and removed the tabs. Do you mind reviewing? We were hoping to get this included in your next release!

enykeev commented:
Thanks! I'll give it a look first thing tomorrow morning.

enykeev commented:
Here is few additions from me. @leleSahil please take a look and tell me if it works for you.

leleSahil commented:
These look good to me! I'm sorry for the late response, I didn't see that you had responded with additional changes. Is it still possible to get this into the most recent release?

enykeev commented:
Yeah, this will be the part of 2.3.
